src/UserSystem/User/User.cs

```diff
@@ -1,28 +0,0 @@
-public abstract class User{
```

Comment:
I noticed you delete this class. Can you explain me why? I don't see any other class related to a User, Patron or Staff. Besides, you are using this class into the repository too.